components/ad/smartbanner/src/index.js

```diff
@@ -49,14 +51,13 @@ function AdSmartbanner({
           </div>
         )}
       </div>
-      <button className={`${baseClass}-buttonInstall`} onClick={onClick}>
-        {buttonText}
-      </button>
+      <Button onClick={onClick}>{buttonText}</Button>
     </div>
   )
 }
 
 AdSmartbanner.propTypes = {
+  button: PropTypes.func,
```

Comment:
It is better to use `PropTypes.node` since it accepts not only functions but: numbers, strings, elements or an array

Reply:
Check it out: https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes